_includes/releases/v22.1/v22.1.0-alpha.2.md

```diff
@@ -0,0 +1,640 @@
+
+<h3 id="v22-1-0-alpha-2-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Client certificates may now be provided for the `webhook` [changefeed sink](../v22.1/changefeed-sinks.html). [#74645][#74645]
+- CockroachDB now redacts more potentially sensitive URI elements from changefeed job descriptions. This is a breaking change for workflows that copy URIs. As an alternative, the unredacted URI may be accessed from the jobs table directly. [#75174][#75174]
+- Changefeeds now outputs the topic names created by the Kafka sink. Furthermore, these topic names will be displayed in the [`SHOW CHANGEFEED JOBS`](../v22.1/show-jobs.html#show-changefeed-jobs) query. [#75223][#75223]
+- [Backup and restore](../v22.1/take-full-and-incremental-backups.html) jobs now allow encryption/decryption with GCS KMS [#75750][#75750]
+- [Kafka sinks](../v22.1/changefeed-sinks.html#kafka) support larger messages, up to 2GB in size. [#76265][#76265]
+- Added support for a new SQL statement called `ALTER CHANGEFEED`, which allows users to add/drop targets for an existing changefeed. The syntax of the statement is: `ALTER CHANGEFEED <job_id> \{\{ADD|DROP\} <targets>\}...`
+
+    There can be an abritrary number of `ADD` or `DROP` commands in any order. For example:
+
+    ~~~ sql
+    ALTER CHANGEFEED 123 ADD foo,bar DROP baz;
+    ~~~
+
+    With this statement, users can avoid going through the process of altering a changefeed on their own, and rely on CockroachDB to carry out this task. [#75737][#75737]
+- Changefeeds running on tables with a low [`gc.ttlseconds`](../v22.1/configure-replication-zones.html#gc-ttlseconds) value now function more reliably due to protected timestamps being maintained for the changefeed targets at the resolved timestamp of the changefeed. The frequency at which the protected timestamp is updated to the resolved timestamp can be configured through the `changefeed.protect_timestamp_interval` [cluster setting](../v22.1/cluster-settings.html). If the changefeed lags too far behind such that storage of old data becomes an issue, cancelling the changefeed will release the protected timestamps and allow garbage collection to resume. If `protect_data_from_gc_on_pause` is unset, pausing the changefeed will release the existing protected timestamp record. [#76605][#76605]
+- Added support to the `ALTER CHANGEFEED` statement so that users can edit and unset the options of an existing changefeed. The syntax of this addition is the following:
+
+    ~~~ sql
+    ALTER CHANGEFEED <job_id> SET <options> UNSET <opt_list>
+    ~~~
+
+    [#76583][#76583]
+- Users may now alter the sink URI of an existing changefeed. This can be achieved by executing `ALTER CHANGEFEED <job_id> SET sink = '<sink_uri>'` where the sink type of the new sink must match the sink type of the old sink that was chosen at the creation of the changefeed. [#77043][#77043]
```

Comment:
```suggestion
- Users can now alter the sink URI of an existing changefeed. This can be achieved by executing `ALTER CHANGEFEED <job_id> SET sink = '<sink_uri>'` where the sink type of the new sink must match the sink type of the old sink that was chosen at the creation of the changefeed. [#77043][#77043]
```